docs/hoon/twig/wut-test/hep-case.md

```diff
@@ -11,13 +11,23 @@ union, with no default.
 ## Expands to
 
 ```
+:loop
 :ifno  q
   :lost 
 :if  :fits(p.i.q p)
   q.i.q
 :moar(q t.q)
 ```
 
+```
+|-
+?.  q
+  :lost
+?:  ?=(p.i.q p)
+  q.i.q
+$(q t.q)
+```
+
 ## Syntax
```

Comment:
`:lost` is another non-existent `twig` expansion; it looks to me like `[%lost]` is expanded in the AST, which signals the compiler to actually throw `mint.lost`, but there's no twig to signify that, and, therefore, no valid expansion